src/replay_stage.rs

```diff
@@ -317,6 +332,12 @@ impl ReplayStage {
                         // We've reached the end of a slot, reset our state and check
                         // for leader rotation
                         if max_tick_height_for_slot == current_tick_height {
+                            let bank_id = current_slot.unwrap();
```

Comment:
B/c we don't skip processing entries for slots for which we are the leader, there's a potential race here where current_tick_height == max_tick_height_for_slot, because the TPU is ticking, but we haven't actually processed all the blobs for the leader slot. Then on the next slot we will get BlobVerification errors because we skipped blobs.